plonky2x/src/frontend/builder/mod.rs

```diff
@@ -36,8 +38,13 @@ pub struct CircuitBuilder<L: PlonkParameters<D>, const D: usize> {
     pub(crate) hints: Vec<Box<dyn HintGenerator<L, D>>>,
     pub(crate) async_hints: Vec<AsyncHintDataRef<L, D>>,
     pub(crate) async_hints_indices: Vec<usize>,
-    pub sha256_requests: Vec<Vec<Target>>,
-    pub sha256_responses: Vec<[Target; 32]>,
+
+    // Right now we only have 2 accelerator, so just have
```

Comment:
Please use full sentences with proper capitalization and end all comments with periods.

In addition, please fill out comments to 100 columns instead of using arbitrary line breaks.